lib/rdoc/parser/c.rb

```diff
@@ -823,10 +814,10 @@ def find_override_comment(class_name, meth_obj)
     comment = if @content =~ %r%Document-method:
                                 \s+#{class_name}#{prefix}#{name}
                                 \s*?\n((?>.*?\*/))%xm then
-                "/*#{$1}"
+                "/*\n#{$1}"
               elsif @content =~ %r%Document-method:
```

Comment:
Can we also merge the condition like: `if @content =~ regexp1 || @content =~ regexp2`